File: Zestawy Zadań/C++/4 - Tablice/Zestaw 2 zadania/4.2.13 - tablica struktur zakupy z produktami.cpp
#include <iostream>
#include <string>

using namespace std;

struct produkt
{
    string nazwa;
    float cena;
    int lsztuk;
};


int main()
{
    produkt tab[10];
    float sumaprod = 0;
    for (int i = 0; i < 2; i++)
    {
        cout << "Podaj nazwe produktu: ";
        cin >> tab[i].nazwa;
        cout << "Podaj cene: ";
        cin >> tab[i].cena;
        cout << "Podaj liczbe sztuk: ";
        cin >> tab[i].lsztuk;
        sumaprod += tab[i].cena * tab[i].lsztuk;
    }
    cout << "Nazwa:" << "\t" << "\t" << "Cena: " << "\t" << "\t" << "Ilosc sztuk: " << endl;
    for (int i = 0; i < 10; i++)
    {
        cout << tab[i].nazwa << "\t" << "\t" << tab[i].cena << "\t" << "\t" << tab[i].lsztuk << endl;
    }
    cout << "Kosz calego zakupu: " << sumaprod << endl;
    return 0;
}

File: Zestawy Zadań/Python/4 - Tablice/Zestaw 2 zadania/4.2.13 - tablica struktur zakupy z produktami.py
class produkt:
    def __init__(self, nazwa, cena, lsztuk):
        self.nazwa = str(nazwa)
        self.cena = float(cena)
        self.lsztuk = int(lsztuk)

sumaproduktow = 0
tab = []

for i in range(10):
    print("Podaj nazwe produktu: ")
    nazwa = str(input())
    print("Podaj cenę produktu: ")
    cena = float(input())
    print("Podaj liczbe sztuk produktu: ")
    lsztuk = int(input())
    tab.append(produkt(nazwa, cena, lsztuk))
    sumaproduktow += cena * lsztuk

for produkt in tab:
    print("Nazwa: ", produkt.nazwa, "\t" , "Cena: ", produkt.cena , "\t", "Liczba sztuk: ", produkt.lsztuk)

print("Suma cen zakupów: ", sumaproduktow)